report_pre_thesis/src/sections/moc.tex

```diff
@@ -78,7 +117,10 @@ \subsubsection{Independent Component Analysis}\label{sec:ica}
 
 ICA yields independent source components and the affiliated mixing matrix which illustrates how the independent sources are combined to form the observed spectral data.
 
-After the extraction of independent components, the key task is to associate each independent component with a single elemental emission line. This involves examining the elements' emission lines and determining the ICA scores, which are then utilized to derive a calibration curve relating the ICA score to the composition.
+After extracting the independent components, the key task is to associate each independent component with a specific elemental emission line. 
+This process involves examining the emission lines of the elements and evaluating the ICA scores. 
+These scores reflect the correlation between each Independent Component and the entire spectrum of wavelengths. 
```

Comment:
We may need something more precise here. It could read as finding the correlation between the IC and the actual wavelength (i.e. 240.811), but it's actually a correlation to the values we see at a given wavelength, no?

Reply:
No, it is actually the correlation to wavelength. 
So the idea is, that you correlate the IC to a wavelength and how much they are correlated is indicated by a score.